index.html

```diff
@@ -0,0 +1,330 @@
+          </div>
+        </nav>
+        <!-- End menu section -->
+      </div>
+      <!-- End container -->
+    </header>
+    <!-- End header section -->
+    <!-- Start main section -->
+    <main>
+      <!-- Start hero area section -->
+      <section class="hero-area">
+        <!-- Start container -->
+        <div class="container">
+          <!-- Start hero main content section -->
+          <div class="hero-main-content">
+            <h1>I'm Nasir Glad to see you!</h1>
+            <p>
+              I'm a software developer! I can help you build a project, feature
+              or website look through some of my work and experience! If you
+              like what you see and have a project you need coded, don't
+              hesitate to contact me.
+            </p>
+          </div>
+          <!-- End hero main content section -->
+          <!-- Start hero social section -->
+          <div class="hero-social">
+            <p>Let's connect</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" rel="noopener noreferrer">
+                  <i class="fa-brands fa-twitter"></i>
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" rel="noopener noreferrer">
+                  <i class="fa-brands fa-linkedin-in"></i>
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" rel="noopener noreferrer">
+                  <i class="fa-solid fa-m"></i>
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" rel="noopener noreferrer">
+                  <i class="fa-brands fa-github"></i>
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" rel="noopener noreferrer">
+                  <i class="fa-regular fa-hand-peace"></i>
+                </a>
+              </li>
+            </ul>
+          </div>
+          <!-- End hero social section -->
+        </div>
+        <!-- End container -->
+      </section>
+      <!-- End hero area section -->
+      <!-- Start portfolio section -->
+      <section id="portfolio">
+        <div class="container">
+          <div class="portfolio-card">
+            <div class="portfolio-thumbnail">
+              <img src="images/work1.jpg" alt="Unplugged Retreat" />
+            </div>
+            <div class="portfolio-content">
+              <h2>Tonic</h2>
+              <ul class="list-items">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
```

Comment:
- [ ] Let's focus on this card as the case study. I am not able to access some of the stylings since there was an inheritance of colors from other sections. As the previous reviewer suggested, kindly stick to the Figma design as much as possible. This is particularly on the colors. The colors mostly used on this card, include #172B4D, #344563, #7A869A. Please modify the colors to match the Figma design. This will make our project more professional.

Figma design
![image](https://user-images.githubusercontent.com/99394374/228249480-eb20d3bd-e160-4f10-a05d-9567723b3e82.png)

Your design
![image](https://user-images.githubusercontent.com/99394374/228249558-3e7b8877-face-4922-8972-6279b1fdfe68.png)